routes/schemas.js

```diff
@@ -47,9 +47,55 @@ module.exports = {
 		// we want it to be stored in MongoDB as a number
 		validUntil: Joi.number().required(),
 		creator: creator(cfg),
+		created: Joi.number(),
+		ipfs: Joi.string().regex(/^ipfs:/),
+		title: Joi.string(),
+		description: Joi.string(),
+		archived: Joi.boolean(),
+		modified: Joi.number(),
 		spec: Joi.object({
```

Comment:
let's change this to enforce the rules for `validators` and `withdrawPeriodStart` but allow *any other properties*; and let's move the adunits schema into the market

Reply:
done